js/gloss-witness.js

```diff
@@ -258,6 +284,100 @@ window.onload = async () => {
  */ 
 addEventListener('deer-view-rendered', addButton)
 
+/**
+ * After a filterableListItem_glossSelector loads, we need to determine what to do with its 'attach' button.
+ * In create/update scenarios, this will result in the need to click a button
+ * In loading scenarios, if a text witness URI was supplied to the page it will have a gloss which should appear as 'attached'.
+ */ 
+function addButton(event) {
```

Comment:
Is this a complete copy-paste? Do we need it in both places?

Reply:
We do because it relies on global variables from both places.  It requires a refactor to be shared in shared.js